utils/copyright_variables.js

```diff
@@ -1,30 +1,57 @@
-export const panelCopy = {
-    title: 'our 3D communication and interaction platform is the solution for Associations with a distributed member base and operations'
-}
-
-export const traditionCopy = {
-    title: 'Are you looking to:',
-    list: [
-        'Amplify member engagement',
-        'Boost retention',
-        'Attract new members'
-    ]
-}
-
-export const solutionCopy = {
-    headerLeft: 'We’re redefining what it means to meet up virtually',
-    headerRight: 'All it takes is your laptop',
-    list: [
-        'Immersive 3D environments',
-        'Dynamic spatial audio',
-        'High-quality video'
-    ],
+export const associationCopy = {
+    panel: {
+        title: 'our 3D communication and interaction platform is the solution for Associations with distributed members and operations'
+    },
+    tradition: {
+        title: 'Are you looking to:',
+        list: [
+            'Amplify member engagement',
+            'Boost retention',
+            'Attract new members'
+        ]
+    },
+    solution: {
+        headerLeft: 'We’re redefining what it means to meet up virtually',
```

Comment:
"The best way to meet in a more human way"

telling them that we are redefining what it means to meet up virtually doesn't sound like a solution